src/pages/Staking/index.tsx

```diff
@@ -336,7 +336,10 @@ const Staking = (): JSX.Element => {
   );
 
   React.useEffect(() => {
-    setBlockLockTimeExtension(Number(lockTime));
```

Comment:
Think I tend to agree, but I'm going to leave it as it is for now. `lockTime` is used quite often in the component in various forms (e.g. `LOCK_TIME`, `extendLockTimeSet`) and since Any change would need to be applied consistently, and with us being so close to launch I'm concerned I might introduce a regression :)